app/mailers/contact_mailer.rb

```diff
@@ -1,13 +1,37 @@
 # encoding: UTF-8
+require 'digest/sha2'
 class ContactMailer < ActionMailer::Base
   default from: 'Spindelman <svarainte@fsektionen.se>', parts_order: ["text/plain", "text/html"]
   default subject: 'Meddelande via Fsektionen.se'
 
-  def contact_email(name, email, msg, kontakt, copy)
-    @name, @email, @msg, @kontakt = name, email, msg, kontakt
-    if @name.present? && @email.present? && @msg.present? && kontakt.present?
-      recipients = copy == '1' ? [@kontakt.email, email] : @kontakt.email
-      mail from: %(#{@name}   <#{@email}>), to: recipients
+  def mail(contact)
+    set_message_id
+    @contact = contact
+
+    if @contact.present?
```

Comment:
@jforberg Does this look like an okay mail setup? 
With the `default from:` at row 4.